### PR TITLE
Provisioner e2e tests

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

- Add a pipeline for tests from https://github.com/kyma-project/control-plane/pull/2886

**Related issue(s)**
https://github.com/kyma-project/test-infra/issues/7662
https://github.com/kyma-project/control-plane/pull/2886

### CODE REVIEW

**Comment by kasiakepka** [comment]
@VOID404 any update on this PR?

**Comment by VOID404** [comment]
@kasiakepka all the related PRs (and status) can be found here: https://github.com/kyma-project/test-infra/issues/7662

File: prow/jobs/control-plane/components/provisioner/provisioner-integration-test.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+presubmits: # runs on PRs
+  kyma-project/control-plane:
+    - name: pre-main-provisioner-e2e-tests
+      annotations:
+        description: "It tests the component's correctness"
+        owner: framefrog
+        pipeline.trigger: "pr-submit"
+        pipeline.type: "integration"
```
Comment (Disper) [inline prow/jobs/control-plane/components/provisioner/provisioner-integration-test.yaml:8]: @kyma-project/prow do you happen to know if that line is required? From what I see all jobs are using the same type.

**Review by Disper — APPROVED** [review]

**Review by Disper — APPROVED** [review]
/meow

**Pull request merged**